src/index.css

```
@@ -1 +1,23 @@
 @import 'tailwindcss';
+
+:root {
```

```suggestion
:root {
```
Måste va @theme {..
Behövs för tailwind theme variables: https://tailwindcss.com/docs/theme#what-are-theme-variables

---

Jag vet men när jag skrev @theme fick jag error och 
To fix the "Unknown at rule @theme" error, you should remove the unsupported @theme block and define your CSS variables inside a :root selector instead.

---

@jont-ctrl har ändrat nu. det kanske var bara en "css lint unknown" grej. hoppas det funkar.